observation_portal/settings.py

```diff
@@ -432,9 +432,11 @@ def get_dramqtiq_broker_url():
     }
 }
 
-# Overrides for Duration calculation functions
-DURATION = {
```

Comment:
I'm putting this other overridable method into the `OVERRIDES` section since it is more generic sounding and we now have two... This will require renaming the override in our lco obs portal project as well.